config.yml

```diff
@@ -58,11 +58,11 @@ weo_2024:
   weo_2024_raw_path: "weo_2024/WEO2024 extended data"
   weo_2024_ext_data_regions_raw_filename: "WEO2024_Extended_Data_Regions.csv"
   weo_2024_ext_data_world_raw_filename: "WEO2024_Extended_Data_World.csv"
-  weo_2024_fig_chptr_3_raw_filename: "WEO2024_Figures_Chapter_03.xlsx"
-  iea_global_ev_raw_path: "iea_global_ev_2024"
-  iea_global_ev_raw_filename: "IEA Global EV Data 2024.csv"
+  weo_2024_fig_chptr_3_raw_full_filepath: 'weo_2024/WEO2024 extended data/WEO2024_Figures_Chapter_03.xlsx'
```

Comment:
in this config, I strive to always separate paths from filenames, which has organizational impact later on in the scripts